### PR TITLE
Add vividus.runStories command

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ export function activate(context: ExtensionContext) {
         progressOnInitialization: true
     };
 
+
```
Comment (valfirst) [inline src/extension.ts:34]: noisy change?

File: vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/command/RunStoriesCommand.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                    "org.vividus.runner.StoriesRunner");
+            ILaunch launch = runner.launch(ILaunchManager.RUN_MODE, null, true);
+            IProcess launchProcess = launch.getProcesses()[0];
+
+            IStreamMonitor streamMonitor = launchProcess.getStreamsProxy().getOutputStreamMonitor();
+            streamMonitor.addListener((text, montor) ->
+            {
+                String messsage = StringUtils.removeEnd(text, "[m");
+                messsage = StringUtils.removeStart(messsage, "\033");
+                messsage = messsage.replaceFirst("^\\[..+?m", "").strip();
```
Comment (valfirst) [inline vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/command/RunStoriesCommand.java:75]: 💩

File: vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/factory/JavaLaunchConfigurationFactory.java [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    }
+
+    private Element readConfigurationXml(String project, String main)
+            throws IOException, ParserConfigurationException, SAXException
+    {
+        try (InputStream inputStream = getClass().getResourceAsStream("/java-launch-config.xml"))
+        {
+            String xmlTemplate = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+            String xml = String.format(xmlTemplate, project, main);
+            DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
```
Comment (valfirst) [inline vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/factory/JavaLaunchConfigurationFactory.java:79]: extract `builder` to constant

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/command/RunStoriesCommand.java (hunk 2 above)
Comment (uarlouski) [reply vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/command/RunStoriesCommand.java:75]: 4pw07x

**Review by valfirst — APPROVED** [review]

**Pull request merged**